src/pages/AuthPage/AuthPage.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { useStore, useDispatch } from "react-redux";
+import axios from 'axios';
+
+import { userSignUp, userLogin } from "../../ducks/authIntoSystem";
+
+import AuthForm from "../../components/AuthForm";
+
+import Img1 from '../../assets/auth-three-people-image.svg';
+import Img2 from '../../assets/auth-games-image.svg';
+import Img3 from '../../assets/auth-robot-image.svg';
+import Img4 from '../../assets/auth-mother-daughter-image.svg';
+import LogoSrc from '../../assets/logo.svg';
+
+import {
+    Auth, AuthBody, Collage, CollageImageWrapper, CollageImage,
+    AuthContent, AuthTitle, AuthFooter, AuthFooterItem, Logo
+} from "./AuthPage.style";
+
+const AuthPage = () => {
+    const store = useStore();
+    const dispatch = useDispatch();
+
+    const fetchData = (formData, authType) => {
+        switch (authType) {
+            case "login":
+                return axios.post(`https://kidslike-v1-backend.goit.global/auth/login`, formData)
+                .then(res => {
+                    dispatch(userLogin(res));
+                })
+            case "register":
```

Comment:
![image](https://user-images.githubusercontent.com/52242019/183218838-2565b899-a634-4962-abc8-85d25a94479c.png)
Pls, take a look at the issue. I entered credentials but no request was sent and a validation error has been received. But, as you can see inputs are filled. It reproduces for both login and registration buttons